duckduckgo/macos-browser
# fix app removal from /Applications in Release

Task/Issue URL: https://app.asana.com/0/1199230911884351/1205203672933066/f

**Description**:

**Steps to test this PR**:
1. Clean. Build in **Debug**, open build log, filter for "Make /Applications" script output - it should contain `rm`, `symlink` etc log
2. Build in **Debug** again, the script log should be empty
3. Build in **Release**, the script build log should contain only "ignoring /Applications/DuckDuckGo.app for Release", /Applications/DuckDuckGo.app should remain in place



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
